test/glific/flows/message_vars_parser_test.exs

```diff
@@ -246,7 +246,8 @@ defmodule Glific.Flows.MessageVarParserTest do
         "category" => "@results.ar_issue_category.category",
         "photo" => "@results.ar_issue_media",
         "othersnumber" => "@results.ar_ir_otherphone",
-        "status" => "@results.ar_issues_exitmenu"
+        "status" => "@results.ar_issues_exitmenu",
+        "storedanswer" => "@results.storedanswer.DetectedResponse"
```

Comment:
didn't understand why this is needed

Reply:
I wrote few test cases which github was showing in the code coverage reports.